sys-apps/yarn/yarn-1.6.0.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Fast, reliable, and secure node dependency management"
+HOMEPAGE="https://yarnpkg.com"
+SRC_URI="https://github.com/yarnpkg/yarn/releases/download/v${PV}/yarn-v${PV}.tar.gz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="!dev-util/cmdtest
+	net-libs/nodejs"
+DEPEND="${RDEPEND}"
+
+S=${WORKDIR}/${PN}-v${PV}
```

Comment:
You're spelling `yarn-v${PV}` literally in `SRC_URI` while using `${PN}-v${PV}` here. Could you make the two consistent? One common way of doing this is creating `MY_P=${PN}-v${PV}` above `DESCRIPTION` and reusing it in both places.